modules/services/agentless-scanning/main.tf

```diff
@@ -301,6 +301,12 @@ resource "aws_kms_key" "scanning" {
   tags                    = var.tags
 }
 
+# Enable key rotation for the KMS key
+resource "aws_kms_key_rotation" "scanning_rotation" {
```

Comment:
@miguelpais Is this a valid resource type? I'm seeing errors invoking this module.